src/util-lua-ssh.c

```diff
@@ -136,3 +192,9 @@ int SCLuaLoadSshLib(lua_State *L)
     luaL_newlib(L, sshlib);
     return 1;
 }
+
+int SCLuaLoadHasshLib(lua_State *L)
+{
+    SCSshEnableHassh();
+    return SCLuaLoadSshLib(L);
```

Comment:
This however does seem odd.

I think it would make more sense to have:
- 2 distinct libraries
- OR, the existing ssh library has an enable haash function which can be called in the init method.

What happens if a hassh method is called but its not enabled?

Reply:
> What happens if a hassh method is called but its not enabled?

print shows `nil`

> the existing ssh library has an enable haash function which can be called in the init method.

Looks the good way to do it to me.
How do I do that ?

Reply:
Like the lua scripts should begin like

```lua

local ssh = require("suricata.ssh")

function init (args)
   local needs = {}
   needs["ssh.hassh"] = tostring(true)
   return needs
end
```

Reply:
Could just do something like:
```
function init ()
  ssh.enable_hassh()
  return {}
end
```
just need to register that function with the sshlib.

Reply:
Looks Even better, will do